frontend/src/scenes/plugins/plugin/PluginCard.tsx

```diff
@@ -127,8 +128,12 @@ export function PluginCard({
                             ) : error ? (
                                 <PluginError error={error} />
                             ) : null}
-
-                            {canInstall ? (
+                            {is_global && (
+                                <Tag color="blue">
+                                    <GlobalOutlined /> Managed by {organization_name}
+                                </Tag>
+                            )}
+                            {user?.organization?.plugins_access_level === PluginsAccessLevel.Root && (
```

Comment:
These shouldn't be hidden by access level alone, but also shown for the install level if it's a plugin for their organisation.